src/item.cpp

```diff
@@ -1744,7 +1766,7 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     }
 
     if (has_flag("FIT")) {
-        ret << _(" (fits)");
+        ret << _(" (fits) ");
```

Comment:
Why this change? Why the change above at line 1634? And below the "(P) " etc. should probably have the space at the front. The other tags have it that way: " (UPS)" and `tagtext` is added after the item name.